File: src/main/java/org/team1/nbe1_3_team01/domain/board/service/converter/DateTimeToStringConverter.java
package org.team1.nbe1_3_team01.domain.board.service.converter;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class DateTimeToStringConverter {

    public static String convert(LocalDateTime dateTime) {
        return dateTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"));
    }
}

File: src/main/java/org/team1/nbe1_3_team01/domain/board/service/converter/DateTimeToStringConverter.kt
package org.team1.nbe1_3_team01.domain.board.service.converter

import lombok.AccessLevel
import lombok.NoArgsConstructor
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

@NoArgsConstructor(access = AccessLevel.PRIVATE)
object DateTimeToStringConverter {
    @JvmStatic
    fun convert(dateTime: LocalDateTime?): String {
        return dateTime?.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"))?:""
    }
}
